snuba/datasets/pipeline/query_pipeline.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC, abstractmethod
+
+import sentry_sdk
+
+from snuba.datasets.plans.query_plan import ClickhouseQueryPlan, QueryRunner
+from snuba.request import Request
+from snuba.web import QueryResult
+
+
+class QueryPipeline(ABC):
```

Comment:
Mind adding a TODO for introducing the pipeline segment so it is clear we are going in that direction ?